App.js

```diff
@@ -0,0 +1,50 @@
+import { View } from 'react-native';
+import Home from './views/Home';
+import Favorites from './views/Favorites';
+import Search from './views/search/Search';
+import { useState } from 'react';
+
+async function getData() {
+  try {
+    const response = await fetch('http://localhost:3000/getTrains/R01N', {
```

Comment:
Can we change the url to a variable for future interoperability in the future when localhost:3000 isn't where our server is?

Reply:
What about in the future when the future is?